internal/templates/template.go

```diff
@@ -23,16 +25,23 @@ func Init(w io.Writer, template, dir string) error {
 	}
 
 	templatePath, ok := templates[template]
+	cloneURL := "https://github.com/jetpack-io/devbox.git"
 	if !ok {
-		return usererr.New("unknown template %q", template)
+		u, err := url.Parse(template)
+		splits := strings.Split(u.Path, "/")
+		if err != nil || u.Scheme != "https" || len(splits) == 0 {
+			return usererr.New("unknown template name or format %q", template)
+		}
+		templatePath = splits[len(splits)-1]
```

Comment:
This won't work if directory is not provided and also ton't work if directory is nested (multiple slashes)